website/docs/cloud-docs/workspaces/dynamic-provider-credentials/hcp-configuration.mdx

```diff
@@ -0,0 +1,110 @@
+
+### Optional Environment Variables
+You may need to set these variables, depending on your use case.
+
+| Variable                                      | Value                                                                                                                                                                                                                                                                      | Notes                                                                                                                                     |
+|-----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| `TFC_HCP_WORKLOAD_IDENTITY_AUDIENCE`          | Will be used as the `aud` claim for the identity token. Defaults to the provider resource name for the current run phase that was derived from the values you provided for `RUN_PROVIDER_RESOURCE_NAME`, `PLAN_PROVIDER_RESOURCE_NAME`, and `APPLY_PROVIDER_RESOURCE_NAME` | Requires **v1.15.1** or later if self-managing agents. This is one of the default `aud` formats that HCP accepts.                          |
+| `TFC_HCP_PLAN_PROVIDER_RESOURCE_NAME`         | The resource name of the workload identity provider that will be used to assume the service principal for the plan phase of a run.                                                                                                                                         | Requires **v1.15.1** or later if self-managing agents. Will fall back to the value of `RUN_PROVIDER_RESOURCE_NAME` if not provided.        |
+| `TFC_HCP_APPLY_PROVIDER_RESOURCE_NAME`        | The resource name of the workload identity provider that will be used to assume the service principal for the apply phase of a run.                                                                                                                                        | Requires **v1.15.1** or later if self-managing agents. Will fall back to the value of `RUN_PROVIDER_RESOURCE_NAME` if not provided.        |
+
+## Configure the HCP Provider
+
+Make sure that you’re not setting values for the `HCP_CRED_FILE`environment variables as these will conflict with the dynamic credentials authentication process.
```

Comment:
Rewrote for active voice, assuming the subject is the HCP provider based on context:
```suggestion
Do not set the `HCP_CRED_FILE` environment variable when configuring the HCP provider, or `HCP_CRED_FILE` will conflict with the dynamic credentials authentication process.
```